odl/operator/default_ops.py

```diff
@@ -298,17 +298,24 @@ def adjoint(self):
         >>> op.adjoint(x)
         rn(3).element([1.0, 4.0, 9.0])
 
-        Multiply by a scalar:
+        Multiply a scalar by vector:
```

Comment:
Something like "Multiply scalars with a fixed vector:" would be more understandable.